code/game/objects/structures/lattice.dm

```diff
@@ -71,9 +71,12 @@
 		return
 	if (istype(C, /obj/item/stack/rods))
 		var/obj/item/stack/rods/R = C
-		if(R.use(2))
+		if(R.get_amount() < 2)
+			to_chat(user, "<span class='notice'>You need at least two rods to form a catwalk here.</span>")

```

Comment:
That's a flimsy catwalk \s

Reply:
it's like that one squid game bridge, you have to stand on the rods to walk through safely